ivadomed/models.py

```diff
@@ -6,6 +6,7 @@
 from torch.nn import init
 from pathlib import Path
 import torchvision.models
+from monai.networks.nets import BasicUNet
```

Comment:
Also need to import UNETR here

Reply:
Fixed in commit version https://github.com/ivadomed/ivadomed/pull/1116/commits/bfccd60402ab3b05b99b356d2c7c35f604441655